blogposts/9-29-21.md

```diff
@@ -187,37 +187,37 @@ In the meantime, check out our [Patreon](https://patreon.com/pretendonetwork) to
 ![](https://i.imgur.com/HK1lXp5.png)
 > Another User's Page
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892939794345246720/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/UserFollowingList.png)
 > Showing Users Following List
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892940579149844510/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/UserSettings.png)
 > User Settings Screen
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892940868674265098/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/Notifications.png)
 > Notifications Screen
 
 ## 3DS Screenshots
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892941422930558986/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/3dsLanding.png)
 > Landing Page
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892941904088539156/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/AllCommunities.png)
 > All Communities and Search
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892942081935413268/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/CommunityPage.png)
 > Community Page 
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892942228954161163/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/ReplyPage.png)
 > New Post/Reply Page
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892942546085511188/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/Spoiler.png)
 > Activity Feed with Spoiler Post
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892943312380645386/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/Doodle.png)
 > Drawing Post with Reply
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892943575560638534/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/UserMenu.png)
 > User Menu Screen
 
-![](https://cdn.discordapp.com/attachments/868275684630470697/892943834521161788/unknown.png)
+![](../public/assets/images/blogposts/9-29-21/Notifications3DS.png)
```

Comment:
None of these links will work. The `public` folder is not exposed to the client, only the files/folders *within* the `public` folder

Also we do not use relative links here. You can see the other blog posts for how to do this